chore(*): Release notes for 6-11-2020 release (#344)
firebase/extensions: storage-resize-images/CHANGELOG.md

## Version 0.1.9

feature - skip resizing of vector images (Fixes #326). (#329)

> build - replace `mkdirp-promise` with `mkdirp` because `mkdirp-promise` has been deprecated. (#266)

Reviewer: ```suggestion
build - Replaced `mkdirp-promise` with `mkdirp` because `mkdirp-promise` is deprecated. (PR #266)
```

Reviewer: I'm not sure if `build` is an accepted release note label?

Author: I'm not certain either - should we switch this to 'fixed' to be safe? I'd also buy the argument that this shouldn't be included in the release notes since its not a user-facing change.

Reviewer: Yeah, I'd agree... let's change to `fixed`.

I think we should keep the info here in the release notes b/c these notes also a record of what changed in the extension source code just in general.

Author: Done!

> fixed - replace `mkdirp-promise` with `mkdirp` because `mkdirp-promise` has been deprecated. (#266)

Reviewer: ```suggestion
fixed - Replaced `mkdirp-promise` with `mkdirp` because `mkdirp-promise` is deprecated. (PR #266)
```

> fixed - If the original image is smaller than the max width and height, the extension does not enlarge it or resize it. (Issue #337, PR #338)

Reviewer: Sorry - just realized something....

```suggestion
fixed - If the original image is smaller than the specified max width and height, the extension does not enlarge it or resize it. (Issue #337, PR #338)
```
